drools-lsp-server/pom.xml

```diff
@@ -80,6 +80,26 @@
         </executions>
 
       </plugin>
+
+      <plugin>
+        <artifactId>maven-antrun-plugin</artifactId>
+        <version>1.8</version>
+        <executions>
+          <execution>
+            <phase>install</phase>
+            <goals>
+              <goal>run</goal>
+            </goals>
+            <configuration>
+              <tasks>
+                <mkdir dir="../client/lib" />
+                <copy file="${project.build.directory}/${project.artifactId}-jar-with-dependencies.jar" todir="../client/lib/" />
```

Comment:
Not mandatory, just my POV.
It makes more sense to me that a package copies external dependencies into itself rather than the dependency copying itself into an external package. Among some options, you could use Webpack and the [copy plugin](https://webpack.js.org/plugins/copy-webpack-plugin/) to achieve that in the `client` package.

Reply:
Thank you for the suggestion. I filed a JIRA https://issues.redhat.com/browse/DROOLS-7400 for that.